skeleton/package.json

```diff
@@ -17,10 +19,11 @@
   "devDependencies": {
     "elm": "^0.19.0",
     "elm-format": "^0.8.0",
-    "elm-live": "next",
+    "elm-live": "^3.0.5",
     "elm-test": "beta",
     "gh-pages": "^1.1.0",
     "npm-run-all": "^4.1.2",
-    "rimraf": "^2.6.1"
+    "rimraf": "^2.6.1",
+    "uglify-js": "^3.1.6"
```

Comment:
With Elm 0.19, output js is already optimized. Please remove uglification again ;)